ClothesApp/ClothesApp/Program.cs

```diff
@@ -1 +1,140 @@
-﻿Console.WriteLine("Hello");
\ No newline at end of file
+﻿using ClothesApp;
+using ClothesApp.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+
+await using var db = new ClothesAppContext();
+
+var productsByBrandIdLazy = await GetProductsByBrandIdLazy(db, 1);
+GetBrandsWithProductAmountLazy(db);
+var productsBySectionCategoryLazy = await GetProductsBySectionCategoryLazy(db, 1, 1);
+var completedOrdersByProductIdLazy = await GetCompletedOrdersByProductIdLazy(db, 1);
+var reviewsForProductLazy = await GetAllReviewsForProductLazy(db, 1);
+
+var productsByBrandId = await GetProductsByBrandId(db, 1);
+GetBrandsWithProductsAmount(db);
+var productsBySectionCategory = await GetProductsBySectionCategory(db, 1, 1);
+var completedOrdersByProductId = await GetCompletedOrdersByProductId(db, 1);
+var reviewsForProduct = await GetAllReviewsForProduct(db, 1);
+
+var brand = await db.Brands.FindAsync((long)4);
+brand!.Name = "mark formelle";
```

Comment:
If you don't want to support nullable reference types you can remove this configuration in csproj